app/helpers/i18n_helper.rb

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 
 module I18nHelper
+  def locales_available
+    OpenFoodNetwork::I18nConfig.available_locales.map do |locale|
+      [t('language_name', locale:), locale]
```

Comment:
That's clever. If this is a problem for anyone, we can then add translation keys for all languages but this is great for now.